src/mangosd/CliRunnable.cpp

```diff
@@ -144,6 +144,9 @@ void CliRunnable::operator()()
                 continue;
             }
             sWorld.QueueCliCommand(new CliCommandHolder(0, SEC_CONSOLE, nullptr, command.c_str(), &utf8print, &commandFinished));
+        #ifndef WIN32
+        free(command_str);
```

Comment:
the indentation does not match the rest of the function

Reply:
Indenting aligns? It's intentionally after the if, so it's free'd even if it's nullptr which is ok to do.

Reply:
> It's intentionally after the if

Looks to be inside the `if (command_str != nullptr)` to me though.

Reply:
🤦 I'm clealy an idiot, thanks for the spotting this both!